src/getRedisClient.js

```diff
@@ -25,11 +25,33 @@ async function initializeRedis() {
   }
 }
 
-async function getRedisClient() {
+async function initializeRemoteRedis() {
```

Comment:
I find "local" and "remote" redis a little confusing. Is the local redis for storing name/pubkey pairs, and remote is for the vanish requests? I think this could use some more documentation at least. Maybe even a rename to "nameRedis" and "vanishRequestRedis" or something like that would be good.

And ideally someday we can refactor down to one Redis instance?

Reply:
oh yeah, down in `src/server.js` they have nicer names.

Reply:
I renamed them to getNip05RedisClient and getVanishRequestsRedisClient. We can always get back to it if we ever add more use cases and we don't migrate to the DO db.